content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/csi-volume-fs-group-policy.md

```diff
@@ -0,0 +1,16 @@
+---
+# Removed from Kubernetes
+title: CSIVolumeFSGroupPolicy
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+---
+<!--
+Allows CSIDrivers to use the `fsGroupPolicy` field.
+This field controls whether volumes created by a CSIDriver support volume ownership
+and permission modifications when these volumes are mounted.
+-->
+允许 CSIDrivers 使用 `fsGroupPolicy` 字段。
```

Comment:
(nit)
```suggestion
允许 CSIDriver 使用 `fsGroupPolicy` 字段。
```
It is OK to leave this unchanged if it matches the current text for the feature gates page.